src/components/RoundSelection/RoundSelection.js

```diff
@@ -1,10 +1,65 @@
-import React from 'react';
+import React, { useContext, useEffect } from 'react';
 import './RoundSelection.css';
 import { Option } from '../Option/Option';
+import { AccessTokenContext } from '../../hooks/TokenContext';
+
+const baseURL = process.env.REACT_APP_SPOTIFY_BASE_URL;
 
 const RoundSelection = (props) => {
-    const {setRoute, setRounds} = props;
+    const {setRoute, setRounds, setAlbums} = props;
+    const accessToken = useContext(AccessTokenContext);
     
+    useEffect(() => {
+        const fetchRandomAlbums = async(accessToken) => {
+
+        const queryParam = 'one%'; // to be randomized
+        const offsetParam = 100; // a number between 1 and 2000
```

Comment:
I think we should use 0 as the default offset to get the most relevant search results back. When a data source is effectively unbounded (like search results on a page) limit and offset based pagination is often used to break the responses into chunks, and only request certain parts of the long long list. 

By passing offset:0,limit:5 we are saying to the API: "Give be the first 5 results back (0-5)".
If we pass offset:100, limit:5 we say: "Give my 5 results, from the 100th result (100-105)

The `next` and `previous` keys in the response are the next and previous offsets around the window of data you are fetching and can be used to fetch the prev/next page (or sometimes called window).

Also, offset is sometimes called cursor in some implementations. Especially when it's id based instead of index based (not the case for Spotify API, the pagination is index based here)

Reply:
Yeah I thought it could be the offset, but it's weird that it worked before. Is the `total` value that is returned the total number of results of this search? If yes we can get a number between 0 and total and set that to the offset. Otherwise we'll just settle on 0.

Reply:
`total` is the total number of results yes.

> If yes we can get a number between 0 and total and set that to the offset. Otherwise we'll just settle on 0. 

Not sure I'm following! We should use offset:0 since I believe the results are ordered by relevance. At the front of the results list (offset:0) we have the most relevant results

Reply:
My point was to randomize both the query _and_ offset, but I'll just use offset: 0